crates/net/src/websocket/futures.rs

```diff
@@ -106,16 +107,12 @@ impl WebSocket {
         url: &str,
         protocols: &[S],
     ) -> Result<Self, JsError> {
-        let map_err = |err| {
+        let json = <JsValue as JsValueSerdeExt>::from_serde(protocols).map_err(|err| {
```

Comment:
Does `JsValue::from_serde` not work?

Reply:
@hamza1311 The issue with `JsValue::from_serde` - `JsValue` currently has a method with the same name if you have `serde-serialization` feature enabled. I wanted to avoid confusion which method is used.

Even when the feature is disabled, `cargo check` was complaining that import isn't used even though it's used.